docs/pages/faq/react-native-version-mismatch.md

```diff
@@ -28,7 +28,7 @@ The packager that you're running in your terminal (either with `expo start` or `
 - If this is a bare workflow project, and this error is occurring right after upgrading your React Native version, you should double-check that you've performed each of the upgrade steps correctly.
 
 - Finally:
-  - Clear your cache by running `rm -rf node_modules && yarn cache clean && yarn && watchman watch-del-all && rm -rf $TMPDIR/react-native-packager-cache-* && rm -rf $TMPDIR/metro-bundler-cache-* && expo start -c`
-     - Commands if you are using NPM can be found [here.](clear-cache-osx-linux)
+  - Clear your bundler caches by running `rm -rf node_modules && yarn cache clean && yarn && watchman watch-del-all && rm -rf $TMPDIR/haste-map-* && rm -rf $TMPDIR/metro-cache && expo start --clear`
```

Comment:
should probably default to npm, people using yarn will figure out to change it to yarn if needed whereas people who use npm might not even know what yarn is